metashare/api/serializers.py

```diff
@@ -287,6 +283,15 @@ class Meta:
     def get_has_unsaved_changes(self, obj) -> bool:
         return bool(getattr(obj, "unsaved_changes", {}))
 
+    def validate(self, data):
+        if ScratchOrg.objects.filter(
+            task=data["task"], org_type=data["org_type"]
+        ).exists():
+            raise serializers.ValidationError(
+                _("A ScratchOrg of this type already exists for this task.")
+            )
+        return data
```

Comment:
Just keep in mind, I would guess that at _some_ point in the future we'll want to allow for multiple reviewers on one task, each with their own org.